modules/sf_h1nobbdde.py

```diff
@@ -0,0 +1,99 @@
+#-------------------------------------------------------------------------------
+# Name:         sfp_h1.nobbd.de
+# Purpose:      Query the the unofficial HackerOne disclosure timeline database to see if our target appears.
+#
+# Author:      Dhiraj Mishra <dhiraj@notsosecure.com>
+# Created:     28/10/2018
+# Copyright:   (c) Dhiraj Mishra
+# Licence:     GPL
+#-------------------------------------------------------------------------------
+
+import sys
+import time
+import datetime
+import re
+import json
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+
+class sfp_h1nobbdde(SpiderFootPlugin):
+    """The unofficial HackerOne disclosure timeline check external vulnerability scanning/reporting service from h1.nobbd.de to see if the target is listed."""
```

Comment:
This needs to be a specific format - please take a look at other modules for examples.